src/schneider_lab_to_nwb/zempolich_2024/zempolich_2024_open_ephys_recording_interface.py

```diff
@@ -0,0 +1,35 @@
+"""Primary class for converting SpikeGadgets Ephys Recordings."""
+from pynwb.file import NWBFile
+import numpy as np
+from typing import Literal
+
+from neuroconv.datainterfaces import OpenEphysLegacyRecordingInterface
+from spikeinterface.extractors import OpenEphysLegacyRecordingExtractor
+
+
+class Zempolich2024OpenEphysRecordingInterface(OpenEphysLegacyRecordingInterface):
```

Comment:
OK. So overriding the recording to make recording modifications work with the class base approach.

I think this is an advantage of using a converter pipe because you don't need to override classes and can have this sort of logic on the conversion script. But this seems like a personal preference of mine for not introducing more classes.

Reply:
Yeah, my biggest consideration here is just that ConverterPipe doesn't have good documentation, so I don't want to pass on a relatively undocumented tool to a lab new to NWB/Neuroconv.

Reply:
Let me know what you think is undocumented so we can fix that.